src/lib.rs

```diff
@@ -959,6 +969,59 @@ impl Ini {
     }
 }
 
+impl Ini {
```

Comment:
this impl is a verbatim scrape of the `impl Ini` block that defines the likes of `write_to`, `write_to_policy`, `write_to_opt`, but replaces each with `std::fmt::Writer` instead of `std::io::Writer`